maxim/tests/test_maxim_core_simple.py

```
@@ -0,0 +1,246 @@
+        self.assertTrue(maxim.prompt_management)
+        # The new cache should be used
+        self.assertEqual(getattr(maxim, "_Maxim__cache"), new_cache)
+        # Should return self for method chaining
+        self.assertEqual(result, maxim)
+
+    def test_enable_prompt_management_starts_sync_thread(self):
+        """Test that enable_prompt_management starts the sync thread."""
+        maxim = Maxim({"api_key": "test-key", "prompt_management": False})
+
+        with patch.object(maxim, "_Maxim__sync_thread", None):
+            result = maxim.enable_prompt_management()
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)